docs/UserGuide.adoc

```diff
@@ -409,50 +410,84 @@ Format: `exit`
 TeachConnect data are saved in the hard disk automatically [even while sharing TeachConnect] after any command that changes the data. +
 There is no need to save manually.
 
-=== Importing the contacts : `import` `[since v1.2]`
+=== Importing the contacts : `import` `[since v1.4]`
 
 TeachConnect allows you to import contacts from a different TeachConnect file by specifying the location of the file. +
 
 Format: `import [TYPE] pathname`
 
 Examples:
-Importing Contacts : `import ./data/importsample.xml`
+* Importing Contacts : `import ./data/importsample.xml`
 
```

Comment:
Wrong formatting. See the preview to see the error.

Also if you want to improve the consistency for examples: it should be in the format:
`import....`
Imports contacts from importsample.xml